SwiftUiDemo/SwiftUiDemo/SwiftUiDemoApp.swift

```diff
@@ -17,12 +17,13 @@ import MoyasarSdk
 /// 10 JPY = 10 JPY (Japanese Yen does not have fractions)
 let paymentRequest = PaymentRequest(
     apiKey: "pk_test_vcFUHJDBwiyRu4Bd3hFuPpTnRPY4gp2ssYdNJMY3",
+    baseUrl: "https://apimig.moyasar.com",
```

Comment:
do we need to update the paymentRequest in the UIKit Demo also ?!

Reply:
Ignore this change, I was testing on staging only